### PR TITLE
feat: about page

### PR DESCRIPTION
**Pull request by aanorbel**
## Proposed Changes
- Add Settings page.
- Add support for launching links in the phone browser.

|.|.|
|-|-|
| Screenshot_20240821_105218 |  Screenshot_20240821_105313 |

### CODE REVIEW

File: composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt [hunk 1]
```diff
@@ -56,6 +57,9 @@ fun Navigation(
                         goToSettingsForCategory = {
                             navController.navigate(Screen.SettingsCategory(it).route)
                         },
+                        sendSupportEmail = {
+                            // TODO(norbel): Send support email
```
Comment (sdsantos) [inline composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt:61]: Maybe we can use the `launchUrl` function as well, but with `mailto:email@example.org`?

File: composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt [hunk 2]
```diff
@@ -99,8 +103,14 @@ fun Navigation(
         ) { entry ->
             val category = entry.arguments?.getString("category") ?: return@composable
             when (category) {
-                PreferenceCategoryKey.SEND_EMAIL.name -> {
-                    // TODO: Implement based on platform
+                PreferenceCategoryKey.ABOUT_OONI.name -> {
+                    val viewModel =
```
Comment (sdsantos) [inline composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt:107]: In these "dumb" views like the About, where no state is read, I'm OK with not having a ViewModel. But it's your call 😉

File: composeApp/src/ooniMain/composeResources/values/strings-organization.xml [hunk 3]
```diff
@@ -19,4 +19,11 @@
     <string name="performance_datausage" translatable="false">5 - 200 MB</string>
 
     <string name="Settings_About_Label">About OONI</string>
+    <string name="Settings_About_Content_Paragraph">The Open Observatory of Network Interference (OONI) is a free software project under The Tor Project that aims to increase transparency of internet censorship around the world.\n\nSince 2012, OONI\'s global community has been measuring networks in more than 200 countries. Some of these measurements serve as evidence of internet censorship.</string>
```
Comment (sdsantos) [inline composeApp/src/ooniMain/composeResources/values/strings-organization.xml:22]: The backspace in `OONI\'s` is showing up in the app. If we switch to the curly version `’` there's no need to escape it.

**Review by sdsantos — APPROVED** [review]

File: composeApp/src/ooniMain/composeResources/values/strings-organization.xml (hunk 3 above)
Comment (aanorbel) [reply composeApp/src/ooniMain/composeResources/values/strings-organization.xml:22]: This is markdown actually. I have to update the view to show the markdown properly.

File: composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt (hunk 1 above)
Comment (aanorbel) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt:61]: Let me update that. I was thinking of a way to add the mail template ( version, device model and other info)

Comment (sdsantos) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt:61]: You can add `cc, bcc, subject and body` as query parameters in the url.

File: composeApp/src/ooniMain/composeResources/values/strings-organization.xml (hunk 3 above)
Comment (sdsantos) [reply composeApp/src/ooniMain/composeResources/values/strings-organization.xml:22]: Ah ok, got it. The curly version would work anyway.

Comment (aanorbel) [reply composeApp/src/ooniMain/composeResources/values/strings-organization.xml:22]: https://github.com/ooni/probe-multiplatform/pull/81

File: composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt (hunk 2 above)
Comment (aanorbel) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt:125]: I will leave the `viewModel` provides an organized way of event handling from the view

**Pull request merged**